.github/pull_request_template.md

```diff
@@ -52,6 +52,9 @@ How to retest this PR or trigger a specific build:
   - for <b>github actions</b> job: 
     add the label `run_fdb`
 
+- for a <b>windows build</b>
+  please add the label `windows_check`
```

Comment:
May I suggest we remove this, as OptaPlanner will not be using this? 

```suggestion
- for a <b>windows build</b>
  please add the label `windows_check`
```

Reply:
Looks ok to me